NEWS

```diff
@@ -0,0 +1,4 @@
+CHANGES IN VERSION 0.99.0
```

Comment:
you can fill this in with the initial release number that you plan to give and a short description? Maybe we should name this 1.0.0?